src/compiler/diagnosticMessages.json

```diff
@@ -751,8 +751,22 @@
         "category": "Error",
         "code": 1235
     },
-
-
+    "'{0}' modifier can only appear on a class or member function declaration.": {

```

Comment:
also i would use "method" every where you have "member function"; i know we have not been consistent, but we can also clean up the other ones that refer to "member functions".